web/src/engine/websites/decorators/Liliana.ts

```diff
@@ -0,0 +1,42 @@
+//For websites similar to MangaSect, Manga347, ManhuaPlus, ManhuaGold,  MangaReader.to,
```

Comment:
There is no [decorator](https://www.typescriptlang.org/docs/handbook/decorators.html) or decorator factory in this file, only some constant strings are exported. Consider moving this to a more appropriate directory.

Reply:
template?

Reply:
In that case i should move MH.ts from decorators to templates, and patch websites accordingly since there is no decorators